src/actions/deploy.ts

```diff
@@ -52,15 +48,6 @@ export async function deploy(
    await checkManifestStability(kubectl, resourceTypes)
    core.endGroup()
 
```

Comment:
this doesn't seem right. If you are removing the routeBlueGreen function call where are you actually routing?

Reply:
I moved the call into deploy. All other actions (promote, reject) call the route methods directly (like routeBGSMI or routeBGService) so I didn't think it made sense to have this "routeBlueGreen" method at all, let alone to have it called outside the call to the deployBlueGreen since that function has preset arguments in the calls that it makes that are suited specifically for deployment. Hence the movement to a "routeForDeploy" function